EvtWeightTools/src/Data_to_MC_CorrectionInterface_0l_2tau_trigger.cc

```diff
@@ -4,29 +4,10 @@
 #include "TallinnNtupleProducer/CommonTools/interface/Era.h"                                    // Era, get_era()
 #include "TallinnNtupleProducer/CommonTools/interface/sysUncertOptions.h"                       // TriggerSFsys
 #include "TallinnNtupleProducer/EvtWeightTools/interface/data_to_MC_corrections_auxFunctions.h" // aux::
-#include "TallinnNtupleProducer/Objects/interface/RecoHadTau.h"                                 // RecoHadTau
-#include "TallinnNtupleProducer/CommonTools/interface/hadTauDefinitions.h"
-
 #include <assert.h>                                                                             // assert()
-
+#include <TMath.h>                                                                          // std::fabs()
```

Comment:
I don't think this line is necessary.

Reply:
this is for std::fabs, if it is not added, there is compilation error

Reply:
In that case prefer STL, ie include `<cmath>`, because that's where `std::fabs()` actually [comes from](https://en.cppreference.com/w/cpp/numeric/math/fabs).